scriptworker/constants.py

```diff
@@ -176,14 +176,16 @@
     # Map scopes to restricted-level
     'cot_restricted_scopes': frozendict({
         'firefox': frozendict({
-            'project:releng:balrog:release': 'all-release-branches',
-            'project:releng:beetmover:release': 'all-release-branches',
+            'project:releng:balrog:server:release': 'all-release-branches',
```

Comment:
This looks good, but:  Do you want to separate nightly/aurora/beta/release/esr on balrog?

We will probably end up using the same creds / server config per scope, unless @mozbhearsum decides we can split the credentials per channel type.  But we can allowlist the various channels each branch can push to: release/releasetest/cdntest etc for release, esr/esrtest/whatever for esr, etc.  In which case, `project:releng:balrog:server:release` would point at `release` instead of `all-release-branches`.